### DISCUSSION TITLE
How to setup a button to edit a line?

### DISCUSSION THREAD

**Discussion by pidanou** [bryanmylee/svelte-headless-table#211] [Q&A]
Hi 

I am trying to create a table where the last cell of each row contains a dropdown menu. 

In that menu, I have an option that is "Edit", it should transform all the cells to inputs.

Is there an easy way to pass that event to all the cells of a row?

Thank you very much

**Comment by nula1337** [comment]
Check the example on shadcn-svelte websites. It includes exactly what you want: https://www.shadcn-svelte.com/docs/components/data-table#row-actions

**Reply by juliarnasution** [reply]
it's not working in svelte 5

**Reply by nula1337** [reply]
Yeah, check #169. shadcn-svelte started using tanstack table for Svelte 5, you can again check their docs.
